src/Microsoft.AspNetCore.ResponseCaching/Interfaces/IResponseCacheStore.cs

```diff
@@ -5,7 +5,7 @@
 
 namespace Microsoft.AspNetCore.ResponseCaching
 {
-    public interface IResponseCache
+    public interface IResponseCacheStore
     {
         object Get(string key);
```

Comment:
Side note: Get and Set will likely need to be async for the distributed/redis scenarios